src/sql/connection.py

```diff
@@ -78,9 +78,7 @@ def get_missing_package_suggestion_str(e):
         module_name, MISSING_PACKAGE_LIST_EXCEPT_MATCHERS.keys()
     )
     if close_matches:
```

Comment:
Why do we need those `\` in here?

Reply:
Cause we want to display the double quote around the dialect name
Ref: [Quotations](https://www.datacamp.com/tutorial/f-string-formatting-in-python#4.1.-quotations)

To show something like this:
```
Perhaps you meant to use driver the dialect: "sqlite"
```